apps/web/test/lib/getSchedule.test.ts

```diff
@@ -929,7 +929,7 @@ describe("getSchedule", () => {
                     date: null,
                   },
                 ],
-                timeZone: Timezones["+5:30"],
+                timeZone: Timezones["+6:00"],
```

Comment:
Changing everything to full hour timezone (+6:00), as with +5:30 one slot got lost. I will take a look at that, if this is an actual bug or the test had some issues